common/config.go

```diff
@@ -0,0 +1,69 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/go-ini/ini"
+)
+
+//
+type DefaultConfigProvider struct {
+	data string
+}
+
+func NewDefaultConfigProvider() *DefaultConfigProvider {
```

Comment:
Is this meant to replace the config file if not present? Currently it still panics without a defaultconfig.ini